code/data_structures/abstract-data-structures/stack/IStack.h

```diff
@@ -0,0 +1,33 @@
+#ifndef ISTACK_H
```

Comment:
This shall be removed as this also exists in the `cpp` folder.

Reply:
Im thinking of adding an other implementation of the IStack -> LinkedStack. Still want me to move it?

Reply:
Okay. Looks good. 👍